mage_ai/cache/block.py

```diff
@@ -30,7 +30,16 @@ async def initialize_cache(
     ) -> 'BlockCache':
         repo_path = repo_path or get_repo_path(root_project=root_project)
         cache = self(repo_path=repo_path)
-        if replace or not cache.exists():
+
+        # Replace the cache if it is older than 2 weeks (the default if days_ago not passed in).
+        cache.remove_old_cache()
```

Comment:
Can you only reset cache when replace is True?

Reply:
updated